src/utils.js

```diff
@@ -140,12 +141,12 @@ export const loadImage = (src, isCrossOrigin = false) =>
   });
 
 /**
+ * @deprecated Please use `scatterplot.createTextureFromUrl(url)`
+ *
```

Comment:
End users will get a deprecation notice in their text editor when importing this function.

Reply:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/24403730/124196424-663ca000-da9a-11eb-86da-de8649cf71e5.png">

Reply:
😍